release/resources/module_list.json

```diff
@@ -235,6 +235,9 @@
         },
         {
             "name": "module-ballerinax-snowflake"
+        },
+        {
+            "name": "module-ballerinax-stripe"
```

Comment:
@ayeshLK this should come under `generated_connectors` (L243). Shall we update accordingly?

Reply:
Fixed with [e69a98589328024f2e9538e07c8d7cf67779e638](https://github.com/ballerina-platform/ballerina-library/pull/6847/commits/e69a98589328024f2e9538e07c8d7cf67779e638)